src/CurrentCredits.js

```diff
@@ -15,8 +15,8 @@ createText(){
     return (
       <div id="sum_current">
         <h5>Aktuelle Leistungspunkte</h5>
-         <p>Du hast in diesem Semester {this.props.selectedCoursesCounter} {this.createText()} mit insgesamt {this.props.currentCreditPoints} Leistungspunkten belegt.</p>
-         <p>Dein Studium wird schätzungsweise noch {this.props.remainingSemesters} Semester andauern.</p>
+         <p>Du hast in diesem Semester <b>{this.props.selectedCoursesCounter}</b> {this.createText()} <br /> mit insgesamt <b>{this.props.currentCreditPoints}</b>  Leistungspunkten <br />belegt.</p>
```

Comment:
I know exactly that it is better to use styling classes instead of inline styling. But considering the quantity of text (className+class) that would be added, I decided to insert the style tag "bold" directly into the code.